docs/networks.md

```diff
@@ -3,14 +3,13 @@
 
 Networks in the HaaS have three fields:
 
-- 'creator' represents what project created it; or 'admin' if it was
-  administrator-created.  If there are still networks created by a project,
-  then that project cannot be deleted.  Only the creator has the ability to
+- 'owner' represents what project owns it; or 'admin' if it was
```

Comment:
You can pass in `null` to json, and it will come through as `None` in python.

From `ipython`:

```python
In [10]: foo = {'a':None}

In [11]: foojson = json.dumps(foo)

In [12]: foojson
Out[12]: '{"a": null}'

In [13]: json.loads(foojson)
Out[13]: {u'a': None}
```